cetmix_tower_webhook/readme/CONFIGURE.md

```
@@ -0,0 +1,52 @@
+- headers: dictionary that contain the request headers
+- raw_data: string with the request raw data
+- payload: dictionary that contains the json payload or the GET parameters of the request
+
+**The code returns the result​ variable in the following format:**
+
+```
+result = {"allowed": <bool, mandatory, default=False>, "http_code": <int, optional>, "message": <str, optional>}
+```
+
+eg:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)